views/index.html

```diff
@@ -72,35 +72,26 @@ <h3>Delete issue on <i>apitest</i></h3>
             crossorigin="anonymous"></script>
     <script>
       $(function() {
-        $('#testForm').submit(function(e) {
-          $.ajax({
-            url: '/api/issues/apitest',
-            type: 'post',
-            data: $('#testForm').serialize(),
-            success: function(data) {
-              $('#jsonResult').text(JSON.stringify(data));
-            }
-          });
-          e.preventDefault();
-        });
```

Comment:
Removing this seems to turn off the response handling and instead sends the JSON result as a new page.